ansible/roles/test/files/ptftests/py3/pfc_wd_background_traffic.py

```diff
@@ -37,8 +37,8 @@ def runTest(self):
             src_mac = self.dataplane.get_mac(0, src_port)
             dst_mac = self.dataplane.get_mac(0, dst_port)
             for queue in self.queues:
-                print(f"traffic from {src_port} to {dst_port}: {queue} ")
```

Comment:
why don't Python3 F-string work with this scritp?

Reply:
This ptf test still uses PY2 intead of PY3. In PY2, f-string is not supported.

Reply:
but this scirpt is located in "py3" folder. is it expected?